src/base/_base.scss

```diff
@@ -35,6 +35,7 @@ html {
 body {
   font-family: $font-primary;
   letter-spacing: 0;
+  white-space: pre-line;
```

Comment:
I'm not sure what issue Andrea was having but just tacked Avery's fix on to this PR since they were both so tiny